research-software-directory/RSD-as-a-service
# fix: aggressive word-wrap of markdown content

# Fix aggresive word-wrap on markdown content

Closes #1210

Changes proposed in this pull request:
*   Change word wrap of markdown content.

How to test:
* `make start` to build app and generate test data
* login as rsd-admin
* create new software of edit existing one
* paste following markdown and confirm cell content is not "aggresively" wrapped

```markdown
# Epidemic Network
*by Marek Vlach*

Experimental environment for testing of large array of theoretical conditions for development epidemic event within various quantitative, spatial and connectedness (network structure) aspects.

## Inputs

|**Name**|**Type**|**Description**|
|-|-|-|
|seed|integer|random generator seed number used as `random-seed seed`|
|initial-households|integer|initial count of households.|
|population-coefficient|integer|household size variance coefficient (mean of exponential distribution). unit: people|
|initial-clusters|integer|initial count of clusters/settlements. unit: clusters/settlements|
|search-radius|integer|maximum search distance to establish links. unit: patch-widths|
|max-connections|integer|maximum number of connections (links) per distance range. unit: connections|
|initial-infected|integer|number of initial cases or people agents initialised with the "incubation" stage and a stochastic state of stage 1 counter. unit: people|
|max-daily-contacts|integer|maximum number of connections to be considered contacts each day. unit: connections|
||||

## Outputs

|**Name**|**Type**|**Description**|
|-|-|-|
|pos-spread|integer|Count of positive disease transmission encounter of an individual (passed to other person)|
|neg-spread|integer|Count of negative disease transmission encounter of an individual (not passed to other person)
||||

## Further information

![Interface screenshot](https://raw.githubusercontent.com/Archaeology-ABM/NASSA-modules/main/2022-Vlach-001/netlogo_implementation/documentation/Epidemic_Network_Model_v1%20interface.png)

See full list of documentation resources in [`documentation`](https://github.com/Archaeology-ABM/NASSA-modules/blob/main/2022-Vlach-001/documentation/tableOfContents.md).
```

## Example fixed word wrap
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/bcf9a7b7-6307-4034-bb38-8ce7df2fc744)


PR Checklist:
